integration_tests/e2e/induction/updateInPrisonInterests.cy.ts

```diff
@@ -0,0 +1,78 @@
+import Page from '../../pages/page'
+import InPrisonWorkPage from '../../pages/induction/InPrisonWorkPage'
+import InPrisonWorkValue from '../../../server/enums/inPrisonWorkValue'
+import EducationAndTrainingPage from '../../pages/overview/EducationAndTrainingPage'
+import { putRequestedFor } from '../../mockApis/wiremock/requestPatternBuilder'
+import { urlEqualTo } from '../../mockApis/wiremock/matchers/url'
+import { matchingJsonPath } from '../../mockApis/wiremock/matchers/content'
+
+context('Update in-prison interests within an Induction', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubAuthUser')
+    cy.task('stubGetHeaderComponent')
+    cy.task('stubGetFooterComponent')
+    cy.task('stubPrisonerList')
+    cy.task('stubCiagInductionList')
+    cy.task('stubActionPlansList')
+    cy.task('getPrisonerById')
+    cy.task('getActionPlan')
+    cy.task('stubLearnerProfile')
+    cy.task('stubLearnerEducation')
+  })
+
+  describe('update in-prison work within an Induction', () => {
```

Comment:
If we are splitting these tests per "screen/question"; do we need this inner "describe" ? and can the contents of the nested beforeEach go in the upper level one?

Reply:
I've removed the describe 👍